### ISSUE TITLE
Annoying pop ups "slow down. too many requests to geocaching.com"

### ISSUE THREAD

**Issue by eddiemuc** [cgeo/cgeo#15882]
Multiple support tickets: 270125, 852190, 580388
C-geo version 2024.07.14-RC

Assumingly caused by https://github.com/cgeo/cgeo/pull/15843

Feedback from one of the  RC-users:

Just tried the newest beta 2024.07.14 and found a pretty annoying popup on my way: "slow down. too many requests to geocaching.com". That popup stays in screen (for 30 seconds or so?) even when I terminate c:geo and go to any other app. 

I can not reproduce, but happened when I had Réunion (Fr) as a whole on screen. 

_Originally posted by @Lineflyer in https://github.com/cgeo/cgeo/pull/15843#issuecomment-2234109348_

**Comment by ztNFny** [comment]
Imho the popup is not the issue here but the fact we're triggering those errors. This is likely the cause for many of those "live map not working" issues we have - and imho it's way preferable to have an info about that than to silently fail.

**Comment by eddiemuc** [comment]
> Imho the popup is not the issue here but the fact we're triggering those errors. This is likely the cause for many of those "live map not working" issues we have - and imho it's way preferable to have an info about that than to silently fail.

True, but the end users should not be affected by this IMHO.

**Comment by eddiemuc** [comment]
A third support ticket came in 

Also there's a first social media reaction: https://www.facebook.com/groups/cgeo.fangruppe/permalink/3438921423073080/?mibextid=rS40aB7S9Ucbxw6v

**Comment by moving-bits** [comment]
We probably need to find  different way of presenting that info. Hiding it behind debug mode will effectively make this warning visible to developers only, but current toast is too intrusive, as it seems to happen so frequently.

Anyway, any adjustments should be targeted to `release`, as current beta version is being built from there.

**Comment by alan666notb** [comment]
I had this problem while out caching today.
I was collecting a set of FTFs and missed one as it didn't appear on my live map.

Back home, when I realised what had happened, I tried to provoke the live map into loading the missing cache by panning around and zooming in/out at its location.
I saw lots of "Slow down" popups, but I could not make the cache load.

Eventually, I had to load it using the search page and typing it GC number in.

**Comment by alan666notb** [comment]
OK - something really odd just happened regarding the cache in my previous post.

The cache was retracted by the reviewer, reason: "Cache Not Showing On Map "
It was then published again (With the same GC code).

So I guess that was a red-herring with respect to the "Slow down" problem.

**Comment by fm-sys** [comment]
> > True, but the end users should not be affected by this IMHO.
> 
> The whole point is that end users get notified. This error is the reason for many of the "Why is c:geo not showing any caches on live map?" problems.

That's true. However seeing that toast permanently makes cgeo unusable as well.

90% of the cases when the toast is shown need to be fixed first before rolling it out to beta users, and for the remaining 10% we can then show the toast.

For now, we should hide that toast for beta/release if we don't want to upset our users, until we e.g. improved the amount of requests while moving the livemap. There is probably a clear issue in the refresh algorithm, and dozens of beta users reporting "I see many annoying messages at the live map" doesn't help us to tackle the issue down either.

**Comment by moving-bits** [comment]
To my understanding this is resolved meanwhile by showing a notification symbol on demand. Feel free to reopen if need be.
